openmetadata-service/src/main/java/org/openmetadata/service/events/EventFilter.java

```diff
@@ -45,21 +47,25 @@ public EventFilter(OpenMetadataApplicationConfig config) {
   }
 
   private void registerEventHandlers(OpenMetadataApplicationConfig config) {
-    try {
+    if (nullOrEmpty(config.getEventHandlerConfiguration())) {
       Set<String> eventHandlerClassNames =
           new HashSet<>(config.getEventHandlerConfiguration().getEventHandlerClassNames());
       for (String eventHandlerClassName : eventHandlerClassNames) {
-        @SuppressWarnings("unchecked")
-        EventHandler eventHandler =
-            ((Class<EventHandler>) Class.forName(eventHandlerClassName))
-                .getConstructor()
-                .newInstance();
-        eventHandler.init(config);
-        eventHandlers.add(eventHandler);
-        LOG.info("Added event handler {}", eventHandlerClassName);
+        try {
+          @SuppressWarnings("unchecked")
```

Comment:
should this supresswarnings be at the method level, rather than at code block level?

Reply:
yes @harshach , addressed :)